src/ScanCommand.php

```diff
@@ -88,7 +108,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $crawlLogger->setOutputFile($input->getOption('output'));
         }
 
-        Crawler::create([RequestOptions::TIMEOUT => $input->getOption('timeout')])
+        $clientOptions = [
+            RequestOptions::TIMEOUT => $input->getOption('timeout'),
+            RequestOptions::VERIFY => ! $input->getOption('skip-verify'),
+            $input->getOption('options'),
```

Comment:
This seems off. If there are multiple options, won't this be a nested array? Does Guzzle use that nested array?